chaincode/compute_plan.go

```diff
@@ -370,6 +370,10 @@ func cancelComputePlan(db *LedgerDB, args []string) (resp outputComputePlan, err
 		return outputComputePlan{}, err
 	}
 
+	if computeplan.Status == StatusCanceled {
```

Comment:
you should also return early if `computeplan.Status` is either `StatusDone` or `StatusFailed` because it means that no tuples can be updated.

Reply:
done